packages/recipes/src/createRuntimeFn.ts

```diff
@@ -62,3 +61,8 @@ export const createRuntimeFn =
 
     return className;
   };
+
+  runtimeFn.variants = Object.keys(config.variantClassNames);
```

Comment:
How about we make `variants` a function instead? That way we only do this work when requested.